notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.        model.load_state_dict(torch.load(args.finetuned_model_path, map_location="cpu"))
</pre><p>how model used here? looks like some actions that does not need</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>On this line, I'm facing the following error:</p><pre class="ql-syntax" spellcheck="false">RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory
</pre><p>PyTorch: 2.0.1+cu117 (installed from this notebook)</p><p> </p>

Reply:
Yes, it shoudn't be here, but it should be executed on the earliest step.